db/schema.rb

```diff
@@ -414,7 +415,9 @@
     t.string "date_format", default: "%m/%d/%Y"
     t.string "signage_convention", default: "inflows_positive"
     t.string "error"
-    t.string "number_format"
+    t.string "currency", default: "USD"
+    t.string "number_format", default: "1,234.56"
```

Comment:
I _think_ the `currency` and `number_format` might be stale from a prior PR correct?  I'm guessing once you clean up migrations this should be resolved.